packages/components/page_template/src/page_header.tsx

```diff
@@ -0,0 +1,28 @@
+import { Flex, FlexProps } from "@buoysoftware/anchor-layout";
+import { Heading } from "@buoysoftware/anchor-typography";
+
+interface OwnProps {
+  title: string;
+}
+
+type PageHeaderProps = OwnProps & Omit<FlexProps, "gridArea">;
+
+export const PageHeader: React.FC<PageHeaderProps> = ({
+  title,
+  ...props
+}): React.ReactElement => {
+  return (
+    <Flex
+      as="header"
+      data-testid="page-header"
+      mb="l"
+      ml="page.gutterX"
+      mt="xxxl"
+      {...props}
+    >
+      <Heading as="h1" gridArea="page-title" size="xl">
+        {title}
+      </Heading>
```

Comment:
Are you waiting to add children as an option until it's needed?

Reply:
That's correct, once I get to that part of infirmary I'll tackle it.